spec/features/venues/update_venue_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature 'Update a venue' do
+  let(:venue) { FactoryGirl.create(:venue) }
+  let(:t_is_accessible_string) { 'T is nearby' }
+
+  before do
+    visit edit_venue_path(venue)
+  end
+
+  scenario 'User edits form correctly' do
+    fill_in('Name', with: venue['name'])
+    fill_in('Capacity', with: venue['capacity'])
```

Comment:
For these, we need to change the values to test an update instead of using the values that are already there.
e.g.,
fill_in('Name', with: 'Updated Name!')